docs/source/concepts/ledger-topologies.rst

```diff
@@ -0,0 +1,179 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _daml-ledger-topologies:
+
+DAML Ledger Topologies
+######################
+
+The Ledger API provides parties with an abstraction of a virtual shared ledger, visualized as follows.
+
+.. image:: ./ledger-model/images/da-ledger-model.svg
+   :width: 80%
+   :align: center
+
+The real-world topologies of actual ledger implementations differ significantly, however.
+The topologies can impact both the functional and non-functional properties of the resulting ledger.
+This document:
+
+1. Provides one useful categorization of the existing implementations' topologies.
+   The categorization is not the only one possible, and it is not always clear-cut.
+   Its main aim is to group the implementations according to their high-level properties.
+
+2. Describes the general ledger properties of each category.
```

Comment:
Having written the above comments: consider replacing the two items with a short summary of the two categories, and add a forward link to Package and Party Management where they are helpful to know.